eng/pipelines/runtime-perf-build-jobs.yml

```diff
@@ -0,0 +1,84 @@
+parameters:
+  runtimeRepoAlias: runtime
+  performanceRepoAlias: self
+  buildConfiguration: []
+  mauiFramework: ''
+
+jobs:
+  - ${{ if containsValue(parameters.buildConfiguration, 'coreclr_arm64_linux') }}:
```

Comment:
This should be simplifiable to an `- ${{ each config in buildConfiguration }}:` where we check the `config` value inside the `parameters` section.